packages/grafana-ui/src/components/Forms/FieldSet.mdx

```diff
@@ -0,0 +1,42 @@
+import { Story, Preview, Props } from '@storybook/addon-docs/blocks';
+import { FieldSet } from './FieldSet';
+
+# FieldSet
+
+Component used to group elements inside a form, equivalent to HTML's [fieldset](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/fieldset) tag.
```

Comment:
```suggestion
Component used to group form inputs, equivalent to HTML's [fieldset](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/fieldset) tag.
```

By `inputs` I mean any form control, regardless if it's radio, select or whatever